modules/ROOT/pages/intro-platform-hosting.adoc

```diff
@@ -0,0 +1,146 @@
+= Anypoint Platform Hosting Options
```

Comment:
Hey Kim! Thanks a lot for the edits! The structure and how to best present this information is something that I have reviewed multiple times with PM. In fact, this is the 3rd draft of this content. I think this approach is the "least bad" of the other drafts. If I were to change the structure this would have to go back through PM review.

Reply:
Also, regarding control and runtime planes, these are concepts that we are introducing to explain groupings of Anypoint Platform features in order to describe where they are hosted and by whom. By hosting, I mean the physical location. These are not terms you will see in the product, though PM/Support/etc. do use them with customers. I don't think we should use them outside of this topic or certainly a small set of topics.

Reply:
In Runtime Manager, we would refer to "deployment targets" instead of runtime plane. In other docs we refer to Management Center or Design Center instead of control plane.